We have no osreldate.h because this header is normally created by
`sys/conf/newvers.sh` but since our libc is based on FreeBSD 8.2
let's just define the symbol.

+++ src/lib/curl/include/curl/curl.h
@@ -47,8 +47,7 @@
 #include <limits.h>
 
 #if defined(__FreeBSD__) && (__FreeBSD__ >= 2)
-/* Needed for __FreeBSD_version symbol definition */
-#include <osreldate.h>
+#define __FreeBSD_version 802000
 #endif
 
 /* The include stuff here below is mainly for time_t! */
